librz/include/rz_il/rz_il_opbuilder_begin.h

```diff
@@ -85,6 +85,7 @@
 #define FMOD(rmode, flx, fly)      rz_il_op_new_fdiv(rmode, flx, fly)
 #define FPOW(rmode, flx, fly)      rz_il_op_new_fpow(rmode, flx, fly)
 #define FMAD(rmode, flx, fly, flz) rz_il_op_new_fmad(rmode, flx, fly, flz)
+#define FEQ(flx, fly)              NOT(OR(FORDER(x, y), FORDER(y, x)))
```

Comment:
@XVilka @Heersin shouldn't this be `FEQ(x0, y0, x1, y1)` ? you can't use the same vals because it will be freed by forder twice, moreover `flx` and `fly` are not used later on in `NOT(OR(FORDER(x, y), FORDER(y, x)))`

Reply:
Good catch. Hopefully should be covered by tests in VFP/NEON PR

Reply:
i can just fix this now.

Reply:
https://github.com/rizinorg/rizin/pull/3571

Reply:
> 

good catch. If it will be a macro in the header, we should DUP(flx) and DUP(fly) to avoid double free.